install_config/install/advanced_install.adoc

```diff
@@ -390,8 +390,27 @@ details.
 |`openshift_clusterid`
 |This variable is a cluster identifier unique to the AWS Availability Zone. Using this avoids potential issues in Amazon Web Service
 (AWS) with multiple zones or multiple clusters. See xref:../../install_config/configuring_aws.adoc#aws-cluster-labeling[Labeling Clusters for AWS] for details.
+
+|`openshift_image_tag`
+|Use this variable to specify a container image tag to install or configure.
+
+|`openshift_pkg_version`
+|Use this variable to specify an RPM version to install or configure.
+
 |===
 
+[WARNING]
+====
+If you modify the `openshift_image_tag` or the `openshift_pkg_version` variables
+after the cluster is set up, then an upgrade can be triggered, resulting in
+downtime.
+
+* If `openshift_image_tag` is set, its value is used for all hosts in
+containerized environments, even those that have another version installed. If
+* `openshift_pkg_version` is set, its value is used for all hosts in RPM-based
```

Comment:
I am not sure "If" should be in the same line of the sentence or in another line on purpose.
@gaurav-nelson

Reply:
Thanks @wmengRH 
Its not on purpose, I'll fix it soon.